### ISSUE TITLE
Install failing with 'package_data must be list of strings'

### ISSUE THREAD

**Issue by aidenrobert** [ljwolf/wampl#6]
Hello!
I'm trying to install this package on a fresh anaconda (4.8.3) environment with python 3.7.6, and am getting the following error with `pip install wes`:

```
Collecting wes
  Using cached wes-0.1.6.tar.gz (199 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/aijo5871/anaconda3/envs/mainenv/bin/python3.7 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-rhrhwpmo/wes/setup.py'"'"'; __file__='"'"'/tmp/pip-install-rhrhwpmo/wes/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-rhrhwpmo/wes/pip-egg-info
         cwd: /tmp/pip-install-rhrhwpmo/wes/
    Complete output (1 lines):
    error in wes setup command: "values of 'package_data' dict" must be a list of strings (got 'colors.json')
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

I also cloned the repo and ran setup.py, and it gives the same:

```
error in wes setup command: "values of 'package_data' dict" must be a list of strings (got 'colors.json')`
```

Curious that it doesn't read it as a list of strings. Any clue what could cause this?

**Comment by drcosquared** [comment]
I was able to resolve this be removing the package data line from the setup.py file. I don't believe it's needed since the path is included in the manifest file

```
from setuptools import setup

setup(name='wes',
      version='0.1.5post2',
      description='Use Wes Anderson Color Palettes in Matplotlib & Seaborn',
      url='https://github.com/ljwolf/wampl',
      author='Levi John Wolf',
      author_email='levi.john.wolf@gmail.com',
      license='CC-BY-SA',
      packages=['wes'],
      install_requires=['matplotlib'],
      include_package_data=True,
      zip_safe=False)
```

**Comment by ljwolf** [comment]
I can also build the package without the `package_data` info. I'll make `0.1.6post1` shortly.

**Comment by ljwolf** [comment]
This should now be addressed with `0.1.7`. Not going to deal with a "real" postrelease since we have plenty of versions before `0.2` :smile:
